pom.xml

```diff
@@ -451,6 +451,49 @@
                 </executions>
             </plugin>
 
+            <!-- Prettier mit git pre-commit hook ausführen -->
+            <plugin>
+                <groupId>com.rudikershaw.gitbuildhook</groupId>
+                <artifactId>git-build-hook-maven-plugin</artifactId>
+                <version>3.4.1</version>
+                <configuration>
+                    <installHooks>
+                        <pre-commit>hooks/pre-commit</pre-commit>
+                    </installHooks>
+                </configuration>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>install</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>2.35.0</version>
+                <configuration>
+                    <formats>
+                        <format>
+                            <includes>
+                                <include>src/**/*.java</include>
+                            </includes>
+                            <prettier>
+                                <devDependencies>
+                                    <prettier>2.8.4</prettier>
+                                    <prettier-plugin-java>2.1.0</prettier-plugin-java>
+                                </devDependencies>
+                                <config>
+                                    <printWidth>120</printWidth>
```

Comment:
Die Indentation beträgt nun 2 Zeichen. Standard in Java, C++, C, ... sind jedoch 4 Zeichen.

Es sollte die Prettierconfig `tabWidth` entsprechend gesetzt werden.

https://prettier.io/docs/en/options.html

Reply:
Du hast Recht, habe ich ergänzt.